tests/unit_tests/lcls_tools/common/measurements/test_screen_beamsize_measurement.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from unittest.mock import MagicMock
+from lcls_tools.common.devices.screen import Screen
+from lcls_tools.common.measurements.screen_profile import (
+    ScreenBeamProfileMeasurement,
+    ScreenBeamProfileMeasurementResult,
+)
+import numpy as np
+
+
+class TestScreenBeamProfileMeasurement(unittest.TestCase):
+    def setUp(self):
+        screen = MagicMock(Screen)
+
+        # create a mock Screen device
+        def mock_get_image(*args):
+            image = np.zeros((100, 100))
+            image[40:60, 40:60] = 255
+            return image
+
+        type(screen).image = property(mock_get_image)
+
+        self.measurement = ScreenBeamProfileMeasurement(device=screen)
+
+    def test_measure(self):
+        result = self.measurement.measure()
+        self.assertIsInstance(result, ScreenBeamProfileMeasurementResult)
+
+        assert result.raw_images.shape == (1, 100, 100)
```

Comment:
I would eliminate many of these tests. Just make sure the total_intensities are correct. That summarizes that the computation along the way is likely to be correct. What you are validating here is that `fit_image(image)` is called, not that `fit_image` is computed correctly (although this does that too with only one test).